src/vxingest/grib2_to_cb/run_ingest_threads.py

```diff
@@ -21,21 +21,21 @@
   "subDoc": "MODEL",
   "subDocType": "HRRR",
   "run_priority": 2,
-  "file_pattern": "%y%j%H%f",
+  "file_mask": "%y%j%H%f",
```

Comment:
Was `file_pattern` supposed to be removed here? It looks like it's still needed by `get_file_list()`.